SPL/SPL_c1.php

```diff
@@ -227,23 +235,29 @@ public function getRealPath(): string|false {}
 
     /**
      * Gets an SplFileInfo object for the file
+     *
+     * @template T of SplFileInfo
```

Comment:
Most of template in the stub have names, like 
- TValue
- TDefault
- TKey
- TPriority
- ...

Maybe there is something better to do here than `T`.